plugins/jobs/server/job_rest.py

```diff
@@ -130,3 +141,19 @@ def updateJob(self, job, log, overwrite, notify, status, progressTotal, progress
     )
     def deleteJob(self, job, params):
         self.model('job', 'jobs').remove(job)
+
+    @access.admin
+    @autoDescribeRoute(
+        Description('Get metadata of all jobs')
```

Comment:
I didn't pay close enough attention to this before, but this method talks about the metadata of the job, though it only returns the types and statuses.  There is a separate metadata field in the job model, which is not returned here, so I think this is confusing.

Better to call it `allJobsTypesAndStatuses` or something along those lines.